### PR TITLE
skywire-cli terminal user interface improvements

### LINKED ISSUE #1385: `skywire-cli` terminal user interface improvements
after my rework of the skywire-cli subcommand structure and flags from a few months ago, subsequent refactors to the formatting of some flags for cli commands has negatively impacted the help menu display, in terms of line wrap, etc.

the skywire-cli help menus will be reformatted to fit again in the default 80x24 terminal window.

Additionally, it should be more easily possible to use the output from one command as the input for another. especially when dealing with public keys or other identifiers which are comprised of long alphanumeric strings.

one example is the `skywire-cli tp` subcommands.

the output of `tp list` could have a flag to only output transport IDs. Then, removing all transports could be something like the following command

```
skywire-cli visor tp rm $(skywire-cli visor tp list --ids)
```

this is one example of numerous possible

### PR DESCRIPTION
**Pull request by 0pcom**
Fixes #1385 

 Changes:	
-	

How to test this PR:

### CODE REVIEW

**Comment by 0pcom** [comment]
Before
image

After
image

Before
image

After
image

Before
image

After
image


Before
image

After
image


Before
image

After
image


Before
image

After
image


Before
image

After
image


Before
image

After
image


Before
image

After
image


Before

After
image


Before
image

After
image


Ready with this one, now

I've left some things commented out which can be implemented after this is merged

File: cmd/skywire-visor/commands/root.go
Comment (mrpalide) [inline cmd/skywire-visor/commands/root.go]: Be good if the behave of this condition not limited to capital words, means set `info` and `Info` works like `INFO`. Currently just `INFO` and `TRACE` and `DEBUG` work well.

**Review by mrpalide — APPROVED** [review]
Approved, just small comment. Will merge after your decision about it.

**Comment by 0pcom** [comment]
Before

image
image



After
image
image

**Pull request merged**
